src/App.tsx

```diff
@@ -15,36 +16,104 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+interface FilterParams {
+  sortField: string;
```

Comment:
```suggestion
  sortField: SortType;
```
specify the type more precisely